pulsar-metadata/src/test/java/org/apache/pulsar/metadata/impl/RocksdbMetadataStoreTest.java

```diff
@@ -35,11 +35,64 @@
 import org.apache.pulsar.metadata.api.MetadataStoreFactory;
 import org.apache.pulsar.metadata.api.Stat;
 import org.testng.Assert;
-import org.testng.annotations.Test;
+import org.testng.annotations.*;
+
+import static org.testng.Assert.assertEquals;
 
 @Slf4j
 public class RocksdbMetadataStoreTest {
 
+    MetadataStore store;
+
+    Path readonlyDataDir;
+    @BeforeClass
+    public void setup() throws Exception {
```

Comment:
This method does not apply to other old test method. Please use a new test class or call it manually in the new tests.

Reply:
Sure, would be change to just call it for readonly mode.